Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
Yeah, that's why using integers for consts is not great IMO. In my projects, I do `MY_CONST = 'my_const'`, way more clear / less collisions